### DISCUSSION TITLE
Remove task

### DISCUSSION THREAD

**Discussion by ghost** [cortinico/ktfmt-gradle#370] [General]
How to remove both tasks which occur during building?

**Comment by simonhauck** [comment]
Do you mean the ktfmtCheck tasks?

**Reply by ghost** [reply]
Yes, when i run `./gradlew clean build`, the first thing to happen is ktfmt runs, i'd like to avoid this behaviour and be the responsable for calling ktfmt.

**Reply by ghost** [reply]
To complement and be more specific. 
I already have a cmd for formatting my code. 
```
tasks.register("fmt") {
    group = "formatting"
    description = "Formats the entire project using ktfmt"

    dependsOn("ktfmtFormat")
}
```
Here what i'd like to disable if possible. I don't want gradle build to have this attached behavior.
![image](https://github.com/user-attachments/assets/90542f5f-956c-4d11-99ad-71f3696c57e5)

**Reply by simonhauck** [reply]
I think this is related to https://github.com/cortinico/ktfmt-gradle/issues/282,

Currently there is no config value, that the _ktfmtCheck_ task does not depend on the _check_ task (which is executed on build). 
You can as a workaround either disable the _ktfmtCheck_ task completly or call it with _-x ktfmtCheck_. This would also skip this task. 

Maybe I will invest soon some time to implement this feature, but no promises :)

**Reply by ghost** [reply]
`-x` will do it, thanks
